components/proof-of-yield/TableHead.tsx

```diff
@@ -0,0 +1,34 @@
+import { Typography } from "@originprotocol/origin-storybook";
+import { PropsWithChildren } from "react";
+import { twMerge } from "tailwind-merge";
+import TitleWithInfo from "./TitleWithInfo";
+
+interface TableHeadProps {
+  info?: string;
+  align?: "left" | "right" | "center" | "justify" | "char";
+  className?: string;
+}
+
+const TableHead = ({
+  info,
+  align,
+  className,
+  children,
+}: PropsWithChildren<TableHeadProps>) => {
+  return (
+    <th
+      align={`${align || "right"}`}
```

Comment:
Similar thoughts on using `align` for td

Reply:
Made this change but `text-`doesn't always work if what I'm aligning is a component. So used `float-` in that case.